lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+  padding-right: $nx-spacing-xs;
```

Comment:
An idea, I'm not really sure this would be better, but it would be more symmetric: what if the tag itself had 8px of side padding on both sides, and then the text had 8px of side margin (or padding) on both sides, and then the icon had 8px of left margin (and 0 right margin).

Reply:
> An idea, I'm not really sure this would be better, but it would be more symmetric: what if the tag itself had 8px of side padding on both sides, and then the text had 8px of side margin (or padding) on both sides, and then the icon had 8px of left margin (and 0 right margin).

Or maybe just have 16px padding on both sides of the tag by default and decrease the right padding to 8px in `.nx-tag--selectable`.  And move all the space between the text and icon into the icon's left margin.

Reply:
> > An idea, I'm not really sure this would be better, but it would be more symmetric: what if the tag itself had 8px of side padding on both sides, and then the text had 8px of side margin (or padding) on both sides, and then the icon had 8px of left margin (and 0 right margin).
> 
> Or maybe just have 16px padding on both sides of the tag by default and decrease the right padding to 8px in `.nx-tag--selectable`. And move all the space between the text and icon into the icon's left margin.

Padding tweaked here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/c13ff544cb89da4ad79a33f6d4e5f8083afb193e